gp-includes/schema.php

```diff
@@ -181,15 +183,15 @@ function gp_schema_get() {
 	 */
 
 	/*
- 	 * The maximum length for the meta_key component of the object_type__meta_key key is limited by the index size limit
+	  * The maximum length for the meta_key component of the object_type__meta_key key is limited by the index size limit
```

Comment:
Only one whitespace after the tab.

Reply:
Fixed in https://github.com/GlotPress/GlotPress-WP/pull/1076/commits/aaf1f070efcc5cb7f040b2d812355ad72d5cc83d